### PR TITLE
Check RIS for bad data

### PR DESCRIPTION
**Pull request by GlazerMann**

### CODE REVIEW

File: tests/phpunit/TemplateTest.php [hunk 1]
```diff
@@ -800,6 +800,25 @@ public function testRIS() {
      $this->assertEquals('Claude E.', $prepared->get('first1'));
      $this->assertEquals('379–423', $prepared->get('pages'));
      $this->assertEquals('27', $prepared->get('volume'));   
+   
+      $text = '{{Cite journal  | TY - JOUR
```
Comment (ms609) [inline tests/phpunit/TemplateTest.php]: Could you add a comment to this test noting why we expect `NULL`, to future-proof the test?
